packer/legacy/jenkins-agent-ubuntu2004-x64.json

```diff
@@ -76,7 +76,7 @@
     {
       "type":"shell",
       "scripts": [
-        "scripts/ubuntu2004/ubuntu2004-agent-setups.sh"
+        "scripts/ubuntu/ubuntu2004-agent-setups.sh"
```

Comment:
As we are moving this to legacy, do we want to modify it?

Reply:
We do not move scripts to legacy at the moment.